e2e/live/end-to-end-test.sh

```diff
@@ -0,0 +1,451 @@
+	ERROR+="ERROR: expected pod $namespace/$podName to exist\n"
+    fi    
+}
+
+# assertPodNotExists checks that a pod with the passed podName and passed
+# namespace DOES NOT exist in the cluster. Waits 20 seconds for pod
+# termination if pod has not finished deleting.
+function assertPodNotExists {
+    local podName=$1
+    local namespace=$2
+
+    kubectl wait --for=delete -n $namespace pod/$podName --timeout=${TIMEOUT_SECS}s > /dev/null 2>&1
+    kubectl get po -n $namespace $podName -o name > $RESULT/podname 2>&1
+    
+    test 1 == $(grep "(NotFound)" $RESULT/podname | wc -l);
+    if [ $? == 0 ]; then
+	echo -n '.'
+    else
+	echo -n 'E'
+	ERROR+="ERROR: expected pod $namespace/$podName to not exist\n"
+    fi    
+}
+
+# printResult prints the results of the previous assert statements
+function printResult {
+    if [ -z $ERROR ]; then
+	echo "SUCCESS"
+    else
+	echo "ERROR"
+    fi
+    echo
+    ERROR=""
+}
+
+# wait sleeps for the passed number of seconds.
+function wait {
+    local numSecs=$1
+
+    sleep $numSecs
+}
+
+###########################################################################
+#  Run tests
+###########################################################################
+
+unset RESOURCE_GROUP_INVENTORY
+
+# Test 1: Basic ConfigMap init
+echo "Testing basic ConfigMap init"
+echo "kpt live init e2e/live/testdata/test-1"
+${BASE}/kpt live init e2e/live/testdata/test-1 > $RESULT/status 2>&1
+assertContains "namespace: test-namespace is used for inventory object"
+assertContains "testdata/test-1/inventory-template.yaml"
+printResult
+
+# Copy the ConfigMap inventory template to the test-2 directory.
+cp -f e2e/live/testdata/test-1/inventory-template.yaml e2e/live/testdata/test-2
+
+# Test 2: Basic kpt live preview
+echo "Testing initial preview"
+echo "kpt live preview e2e/live/testdata/test-1"
+${BASE}/kpt live preview e2e/live/testdata/test-1 > $RESULT/status
+assertContains "namespace/test-namespace created (preview)"
+assertContains "pod/pod-a created (preview)"
+assertContains "pod/pod-b created (preview)"
+assertContains "pod/pod-c created (preview)"
+assertContains "4 resource(s) applied. 4 created, 0 unchanged, 0 configured"
+assertContains "0 resource(s) pruned, 0 skipped"
+printResult
+
+# Test 3: Basic kpt live apply
+echo "Testing basic apply"
+echo "kpt live apply e2e/live/testdata/test-1"
+${BASE}/kpt live apply e2e/live/testdata/test-1 > $RESULT/status
+assertContains "namespace/test-namespace"
+assertContains "pod/pod-a created"
+assertContains "pod/pod-b created"
+assertContains "pod/pod-c created"
+assertContains "4 resource(s) applied. 3 created, 1 unchanged, 0 configured"
+assertContains "0 resource(s) pruned, 0 skipped"
+wait 2
+# Validate resources in the cluster
+# ConfigMap inventory with four inventory items.
+assertCMInventory "test-namespace" "4"
+printResult
+
+# Test 4: kpt live preview of apply/prune
+echo "Testing basic preview"
+echo "kpt live preview e2e/live/testdata/test-2"
```

Comment:
Can you add a comment for "test-2"? It contains the new set of resources for the same package in test-1.